cmd/av/stack_sync.go

```diff
@@ -48,6 +48,7 @@ latest commit to the repository base branch (e.g., main or master) into the
 stack. This is useful for rebasing a whole stack on the latest changes from the
 base branch.
 `),
+	PositionalArgs: cobra.NoArgs,
```

Comment:
This should be `Args`

Reply:
Yes, I had this locally and then pushed the wrong state. >.<
Rewriting history please hold.